src/app/footer/footer.component.scss

```diff
@@ -0,0 +1,215 @@
+@import '../../variables' ;
+
+.footer {
+    flex-shrink: 0;
+    display: block;
+}
+
+.pt-5 {
+    padding-top: 3rem !important;
+}  
+
+.col-md-4 {
+    flex: 0 0 auto;
+    width: 33.33333333%;
+}
+
+.widget-title {
+    margin-bottom: 1.125rem;
+    font-size: 1.0625rem;
+    font-weight: 600;
+}
+
+.widget-list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    &[data-simplebar] {
+        overflow-x: hidden;
+    }
+}
+
+.widget-list-item {
+    margin-bottom: 0.5rem;
+    list-style-type: none;
+    a:hover{
+        color: whitesmoke;
+        font-weight: bold;
+    }
+}
+
+.widget-list-link {
+    display: block;
+    color: rgba(255,255,255,.65);
+    font-size: .875rem;
+    font-weight: normal;
+    text-decoration: none;
+}
+
+.widget{
+    color: #f7f9fd;
+    margin: 0px 0px 24px;
+    padding: 0px 0px 8px;
+    list-style: none;
+    .widget-links{
+        display: inline-block;
+    }
+}   
+.pb-2 {
+   padding-bottom: 0.5rem !important;
+}
+.mb-4 {
+   margin-bottom: 1.5rem !important;
+}
+
+[role=button] {
+    cursor: pointer;
+}
+
+.btn-apple {
+    background-image: url(../../assets/appleicon.png);
+}
+
+.btn-google {
+    background-image: url(../../assets/googleplay.png);
+}
+
+// Footer Back-ground colors
+
+.bg-footer {
+    background-color: $footer-colour;
+}
+
+.bg-footer-lighter{ 
+    background-color: lighten($footer-colour, 25%);
+}
+
+.bg-footer-slighter{
+    background-color: lighten($footer-colour, 20%);
+}
+
+// Footer - Icons at bottom
+
+.text-primary{
+    color: #fe696a !important;
+}
+
+.ps-3 {
+    padding-left: 1rem !important;
+}
+
+.pb-3 {
+    padding-bottom: 1rem !important;
+}
+
+.mb-4 {
+    margin-bottom: 1.5rem !important;
+}
+
+.d-flex {
+    display: flex !important;
+    align-items: center;
+}
+
+.fs-base {
+    font-size: 1rem !important;
+    font-weight: bold;
+    display: block;
+
+    .text-light {
+        color: #fff !important;
+    }
+    .mb-1 {
+        margin-bottom: 0.25rem !important;
+    }
+}
+
+.fs-ms {
+    font-size: 0.8125rem !important;
+    white-space: nowrap;
+}
+
+.mb-0 {
+    margin-bottom: 0 !important;
+}
+
+// Apple and Google Apps
+
+.btn-market {
+    display: inline-block;
+    padding: 0.375rem 0.875rem;
+    padding-left: 2.8125rem;
+    border-radius: 0.375rem;
+    background-position: center left 0.75rem;
+    background-color: #2b3445;
```

Comment:
define the color in variable file

Reply:
@akashsinha13, modified and taking it from variable.scss